pilot/pkg/networking/grpcgen/lds.go

```diff
@@ -270,7 +270,7 @@ func buildRBAC(node *model.Proxy, push *model.PushContext, suffix string, contex
 	for _, policy := range policies {
 		for i, rule := range policy.Spec.Rules {
 			name := fmt.Sprintf("%s-%s-%d", policy.Namespace, policy.Name, i)
-			m, err := authzmodel.New(rule)
+			m, err := authzmodel.New(rule, true)
```

Comment:
Does grpc support this filter before? I am concerned about the compatibility here

Reply:
gRPC doesn't support it either way. Any value here would not work for JWT.